From ae2e19dd96e47a084eac09c3b3095b3bef427415 Mon Sep 17 00:00:00 2001
From: Thierry Strudel <tstrudel@google.com>
Date: Thu, 19 Jul 2018 18:49:15 -0700
Subject: [PATCH 07/29] metricslogger: fix ACTION_BATTERY_CAUSED_SHUTDOWN not
 matching proto

Bug: 111664962
Test: "pixelstats_client -s" uses the right ATOM value
Change-Id: I2944c6d9f79298ce88812fad218552b92afee97a
Signed-off-by: Thierry Strudel <tstrudel@google.com>
---
 libmetricslogger/include/metricslogger/metrics_logger.h | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/libmetricslogger/include/metricslogger/metrics_logger.h b/libmetricslogger/include/metricslogger/metrics_logger.h
index c305db2da..88575be9d 100644
--- a/libmetricslogger/include/metricslogger/metrics_logger.h
+++ b/libmetricslogger/include/metricslogger/metrics_logger.h
@@ -114,7 +114,7 @@ enum {
     FIELD_BATTERY_RESISTANCE_UOHMS = 1448,
     FIELD_BATTERY_CURRENT_UA = 1449,
     FIELD_HARDWARE_LOCATION = 1450,
-    ACTION_BATTERY_CAUSED_SHUTDOWN = 1441,
+    ACTION_BATTERY_CAUSED_SHUTDOWN = 1451,
 };
 
 enum {
-- 
2.17.1

